renovate.json

```diff
@@ -40,23 +37,20 @@
       "enabled": false
     },
     {
+      "groupName": "arrow-rs",
       "matchUpdateTypes": [
         "major",
         "minor",
         "patch"
       ],
-      "groupName": "datafusion and friends",
-      "matchPackageNames": [
-        "arrow{/,}**",
-        "parquet{/,}**",
-        "datafusion{/,}**"
+      "matchSourceUrls": [
+        "https://github.com/apache/arrow-rs"
```

Comment:
just learned this is a thing 👀, adding it to my list of things we can do to simplify our renovate configs.